### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by ethanis** [github/gh-actions-importer#54] [Announcements]
## What's Changed
* CI: add required permissions for publishing unit test results by @simonsanchez in https://github.com/github/gh-actions-importer/pull/53
* Bump NUnit3TestAdapter from 4.3.1 to 4.4.0 in /src by @dependabot in https://github.com/github/gh-actions-importer/pull/51
* Bump Microsoft.NET.Test.Sdk from 17.4.1 to 17.5.0 in /src by @dependabot in https://github.com/github/gh-actions-importer/pull/52
* updates for ga by @JenniferKerns in https://github.com/github/gh-actions-importer/pull/46

## New Contributors
* @simonsanchez made their first contribution in https://github.com/github/gh-actions-importer/pull/53

**Full Changelog**: https://github.com/github/gh-actions-importer/compare/v1.0.4...v1.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/github/gh-actions-importer/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by Sjcbdudnxbx** [comment]
Hi from me and nick
